src/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Grid</title>
+    <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+  </head>
+
+  <body>
+    <div class="containter">
+      <div class="containter__logo">
+        <span class="containter__logo-title">logo</span>
+      </div>
+      <header class="containter__header header">
+        <nav>
+          <ul class="header__list">
+            <li class="header__item">
+              <a class="header__link" href="#">home</a>
+            </li>
+            <li class="header__item">
+              <a class="header__link" href="#">about</a>
+            </li>
+            <li class="header__item">
+              <a class="header__link" href="#">gallery</a>
+            </li>
+          </ul>
+        </nav>
+        <a class="header__link" href="#">login</a>
+      </header>
+      <aside class="containter__sidebar sidebar">
+        <p class="sidebar__menu">menu</p>
+        <div class="sidebar__list">
+          <nav>
+            <ul class="sidebar__list">
+              <li class="sidebar__item">
+                <a class="sidebar__link" href="#">article</a>
+              </li>
+              <li class="sidebar__item">
+                <a class="sidebar__link" href="#">news</a>
+              </li>
+              <li class="sidebar__item">
+                <a class="sidebar__link" href="#">blog</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </aside>
+      <main class="containter__content content">
+        <div class="content__one">
+          <span>content 1</span>
+        </div>
+        <div class="content__two">
+          <span>content 2</span>
+        </div>
+        <div class="content__three">
+          <span>content 3</span>
+        </div>
+        <div class="content__four">
+          <span>content 4</span>
+        </div>
+      </main>
+      <footer class="containter__footer">
+        <p class="containter__footer-title">footer</p>
```

Comment:
Better use <span> as it is not a paragraph

Reply:
ok